postcss.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Can we keep all frontend specific files inside `renderer` folder please? I.e. move `postcss.config.js` to `renderer/postcss.config.js`

Unless tooling like VSCode requires the config file to be in the project root.

What do you think, @jopedroliveira and @juliangruber?

The same comment applies to `tailwind.config.js` below.

Reply:
Agree.

Reply:
The docs refer that those files should be in the root. As I could check there's no clean way of moving it. 
I'll re-visit it later

Reply:
I see.

Quoting from https://vitejs.dev/guide/features.html#postcss
> If the project contains valid PostCSS config (any format supported by [postcss-load-config](https://github.com/postcss/postcss-load-config), e.g. postcss.config.js), it will be automatically applied to all imported CSS.

The docs for postcss-load-config offers many different file names for the configuration, but all of them are in the project root :(

Considering that we already have `vite.config.js` in project root too, I am fine to add PostCSS & Tailwind config files to project root too.